scripts/gen_bytecode.lua

```diff
@@ -10,7 +10,9 @@ function gen_bytecode()
 
         function get_account(name)
             account_key = get_account_key(name)
-            account_data = coroutine.yield(account_key)
+
+            -- 0 is read lock, 1 is write lock
```

Comment:
perhaps consider using something more enum-like? https://unendli.ch/posts/2016-07-22-enumerations-in-lua.html